docs/content/concepts/ops-jobs-graphs/ops.mdx

```diff
@@ -1,42 +1,41 @@
 
-- Derive a data set from some other data sets.
+- Add computed columns to a dataframe.
 - Execute a database query.
 - Initiate a Spark job in a remote cluster.
 - Query an API and store the result in a data warehouse.
 - Send an email or Slack message.
 
-Depending on how you execute a job, Dagster can run all the ops in the job within the same process or isolate them in different processes.
+The op as computational unit enables many useful features for data orchestration:
 
-Ops have several important properties:
-
-- **Inputs and Outputs**: Ops have defined [inputs and outputs](#inputs-and-outputs), which can be optionally typed. These types are validated at runtime.
-- **Configurable**: Ops can be configured, using a strongly typed [configuration system](/concepts/configuration/config-schema).
-- **Emit an Event Stream**: Ops can emit a stream of structured events, such as AssetMaterializations. These events can be viewed in [Dagit](/concepts/dagit/dagit), Dagster's UI tool.
-- **Individually testable**: See the [testing page](/concepts/testing) for more detail.
+- **Flexible execution strategies**: Data pipelines are frequently developed locally and later deployed to production. Ops represent sealed units of logic independent of pipeline execution strategy, making the transition from development to production painless. Collections of ops (i.e. [Jobs](/concepts/ops-jobs-graphs/jobs-graphs)) can be passed to an appropriate [Executor](/deployment/executors) for single-process execution or distribution across a cluster.
+- **Input and output management**: Ops have defined [inputs and outputs](#inputs-and-outputs) analogous to the arguments and return value(s) of a Python function. However, op inputs/outputs can be more richly annotated than Python function arguments. This annotation can instruct Dagster's engines how to validate, cache, serialize, and otherwise manage data as it flows between ops.
+- **Configuration**: Operations in a data pipeline are often parameterized by both upstream data (e.g. a stream of database records) and "configuration" values independent of upstream data (e.g. a "chunk size" of incoming records to operate on). Op configuration values can be defined and annotated in much the same way as inputs and outputs.
+- **Event streams**: Ops emit a stream of [Events](/concepts/ops-jobs-graphs/op-events) during execution. Certain events are emitted by default (e.g. indicating the start of an op's execution), but op authors are additionally given access to an event API. This can be used to report data asset creation or modification (<PyObject object="AssetMaterialization"/>), the result of a data quality check (<PyObject object="ExpectationResult"/>), or other arbitrary information. Event streams can be visualized by Dagster's browser UI [Dagit](/concepts/dagit/dagit). This rich log of execution facilitates debugging, inspection, and real-time monitoring of pipeline execution and data asset production.
```

Comment:
"execution" twice in the last sentence is a tiny bit awkward. Could just leave out the words after "monitoring"?

Reply:
Good catch